source/documentation/common/install/proc-Enabling_the_Red_Hat_Virtualization_Manager_Repositories.adoc

```diff
@@ -152,6 +152,18 @@ ifdef::ovirt-doc[]
 # dnf module -y enable mod_auth_openidc:2.3
 ----
 endif::ovirt-doc[]
+. Enable version 14 of the `nodejs` module:
+[source,terminal,subs="normal"]
++
+----
+# dnf module -y enable nodejs:14
+----
+. Enable the `maven` module:
+[source,terminal,subs="normal"]
++
+----
+# dnf module -y enable maven
```

Comment:
you need to enable maven 3.6 because the default 3.5 is out of support -- https://access.redhat.com/support/policy/updates/rhel-app-streams-life-cycle

Reply:
dnf module -y enable maven:3.6

Reply:
OK, according to the last message from Martin Perina, 
I will write 
dnf module -y enable maven:3.6
for now, until the package dependencies can be resolved (BZ#2089299 [](https://bugzilla.redhat.com/show_bug.cgi?id=2089299) - which is targeted right now for oVirt 4.5.1

Reply:
adjusting the document so that the oVirt version says maven:3.6
and the RHV version says maven:3.5

Reply:
Hiding maven instructions